### DISCUSSION TITLE
Custom load shapes to primary loads

### DISCUSSION THREAD

**Discussion by vmotakat** [GRIDAPPSD/gridappsd-forum#140] [API] [answered]
Hi,
I started my platform with ./run.sh -t develop to update my platform. The feeder I am using don't have any secondary part that means only primary side of feeder with so many loads in the system.

 I loaded custom load shape in to influx db container. I started simulations for one day with offline mode. I observed total load on the simulation results indicated that custom load shapes not able to implement with recent changes.

Can any help me to fix this issue.

@poorva1209

**Comment by afisher1** [comment]
post the model_base.glm if you can

**Reply by vmotakat** [reply]
Its not possible to post gym file as it related to real time feeder of use case-6. Can we have quick look in teams call if possible.

**Reply by poorva1209** [reply]
@vmotakat  You can share dropbox or sharepoint link of that file to Andy if that works.

**Comment by poorva1209** [comment]
@vmotakat Are you using the latest script from gridappsd-data to load custom load data? Did that uploaded without any errors?

**Reply by vmotakat** [reply]
Yes without error I am able to load  custom load data to platform.

<img width="1685" alt="Screen Shot 2022-08-25 at 12 57 44 PM" src="https://user-images.githubusercontent.com/102381298/186746512-9ac3fda2-3feb-4995-a4b2-518154cc996d.png">

**Reply by poorva1209** [reply]
ok it says 'Failed 0 inserts' so it is loading without errors. Can you do 'select * from ieeezipload limit 2' and send the response.

**Comment by vmotakat** [comment]
Hi now issue resolved after the changes suggested by you both. You can close this thank you!!

**Comment by poorva1209** [comment, marked as answer]
This issue was resolved using the latest develop tag of GridAPPS-D platform as of 8/25/2022.
